code/index.d.ts

```diff
@@ -0,0 +1,189 @@
+// Type definitions for code 4.0.0
+// Project: https://github.com/hapijs/code
+// Definitions by: Prashant Tiwari <https://github.com/prashaantt>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "code" {
```

Comment:
could you strip `declare module "code" {`?
https://github.com/Microsoft/types-publisher#fields-in-datadefinitionsjson
> ProperModule: Only top-level import and export declarations. Preferred